scripts/run_e2e_tests_test.py

```diff
@@ -775,60 +603,6 @@ def test_start_tests_in_webdriverio(self):
         run_e2e_tests.main(
             args=['--suite', 'collections'])
 
-    def test_do_not_run_with_invalid_test_suite(self):
```

Comment:
Do we not handle invalid test suites anymore? That seems like something we should keep around

Reply:
We already have a test named `test_main_with_invalid_test_suites` for this. Also this test was added because we were using if condition while running the test in hybrid mode i.e if test present in protractor list run test in protractor else if test is present in webdriverio list then run the test in webdriverio else the suite is invalid as it not present in any of the list. So this test was only targeting the last condition which is removed now.